azure/v12.1.0/README.md

```diff
@@ -0,0 +1,28 @@
+## :zap:  Giant Swarm Release 12.1.0 for Azure :zap:
+
+**If you are upgrading from 12.0.2, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release upgrades all Helm releases managed by Giant Swarm to use [Helm v3.3.0](https://github.com/helm/helm/releases/tag/v3.3.0).
+
+This lets us benefit from the improved security model and keep up to date with the community. We also remove the Tiller deployment from the giantswarm namespace, removing its gRPC endpoint, which reduces operational complexity.
+
+If you are still using Helm 2 then these Helm releases will not be affected. However we encourage you to upgrade to Helm 3. As Helm 2 support ends on November 13th 2020. https://helm.sh/blog/helm-v2-deprecation-timeline/
+
+Below, you can find more details on components that were changed with this release.
+
+**Note before upgrade:**
+
+Please contact your Solution Engineer before upgrading. The upgrade is automated. However, it includes a data migration from Helm 2 release configmaps to Helm 3 release secrets, which we recommend to monitor to ensure safety.
+
+**Note for Solution Engineers:**
```

Comment:
@rossf7 @tomahawk28 @pipo02mix and @yasn77 (since you are moving to Batman :P), I made the changes here. Please check if it makes sense to you (especially Fer and Yasser for guiding customer upgrade)

Reply:
Made my comment in Slack (before I knew about this issue 😸 ) -> https://gigantic.slack.com/archives/C76JX6YLQ/p1599040644057500?thread_ts=1599031196.044300&cid=C76JX6YLQ